assets/src/scripts/components/hydrograph/graph-brush.js

```diff
@@ -70,11 +72,25 @@ export const drawGraphBrush = function(container, store) {
                 selection = [0, layout.width - layout.margin.right];
             }
             brushElem.remove();
+
             const group = svg.append('g').attr('class', 'brush')
                 .attr('transform', `translate(${layout.margin.left},${layout.margin.top})`);
+
             graphBrush.extent([[0, 0], [layout.width - layout.margin.right, layout.height - layout.margin.bottom]]);
+
+            // Creates the brush
             group.call(graphBrush);
+
+            // Fill & round corners of brush handles, set mouseover highlight
+            svg.selectAll('.handle').style('fill', '#022F54').style('opacity', .5)
```

Comment:
I would like to see the style defined in the Sass style sheets. You could put the style for this in https://github.com/usgs/waterdataui/blob/master/assets/src/styles/components/hydrograph/_app.scss. I would also like to stick with USWDS color tokens: https://designsystem.digital.gov/design-tokens/color/system-tokens/. For the transitions below you could add a class during the first transition which will have the style you want for the transition and then remove that class.

Reply:
I created a brush-handle-fill style class.  I tried to also create a separate highlight class and toggle between the two within the transition, but it didn't seem to like that, but I choose two colors from the "Blue warm" category.